README.md

```diff
@@ -14,22 +14,24 @@ Conceptlist in Concepticon: [Key-2016-1310](http://concepticon.clld.org/contribu
 
 
 [![Build Status](https://travis-ci.org/lexibank/ids.svg?branch=master)](https://travis-ci.org/lexibank/ids)
-![Glottolog: 100%](https://img.shields.io/badge/Glottolog-100%25-brightgreen.svg "Glottolog: 100%")
+![Glottolog: 98%](https://img.shields.io/badge/Glottolog-98%25-green.svg "Glottolog: 98%")
 ![Concepticon: 100%](https://img.shields.io/badge/Concepticon-100%25-brightgreen.svg "Concepticon: 100%")
-![Source: 22%](https://img.shields.io/badge/Source-22%25-red.svg "Source: 22%")
+![Source: 20%](https://img.shields.io/badge/Source-20%25-red.svg "Source: 20%")
+![BIPA: 45%](https://img.shields.io/badge/BIPA-45%25-red.svg "BIPA: 45%")
+![CLTS SoundClass: 44%](https://img.shields.io/badge/CLTS%20SoundClass-44%25-red.svg "CLTS SoundClass: 44%")
 
-- **Varieties:** 329
+- **Varieties:** 296
```

Comment:
Do you really want to remove 33 wordlists from the dataset?

Reply:
I certainly don't! I'll check what happened, sorry.

Reply:
The problem is that the form stripping function returns none if the transcription is not in a certain range. So we are missing one other type of transcription apart from latin, cyrillic, or whatsoever here, and this kills the thirty languages. Should be easy to fix by looking into the different transcriptionvalues.

Reply:
That's indeed the problem, I'm looking at the output of the previous `clean_form()` in order to look for the best solution. Unfortunately, given the variety of sources/languages and usages of stuff like brackets, the less worst one is likely to keep everything as it was, in order not to lose any valid information.

Reply:
Yes, the form should be kept as is, if it's not '???' or similar. I think, ther'es simply one more type of trnascription not accounted for.

Reply:
I am cleaning most forms, as in your code, but defaulting to keeping the transcription in the worst cases. The current filter is good enough and there is no point in working through the data before having at least a preliminary orthographic profile (which might solve most of the problems).